### PR TITLE
add example abaplint configuration

### PR DESCRIPTION
**Pull request by larshp**
This will run checks for the `/file-formats/*/*.intf.abap` files according to the configuration file

* No void types(unknown types) are allowed
* Only unused type allowed is "main"
* Keyword casing is upper case
* Plus more, see https://rules.abaplint.org for details

this will catch syntax errors like https://github.com/SAP/abap-file-formats/pull/70/commits/8b08357341470e5a8d9ba8541ead72dcf1acef52 automatically

[vscode extension](https://marketplace.visualstudio.com/items?itemName=larshp.vscode-abaplint), it also works with SCI, but for now its easier just to modify the file on github than cloning into a system

### CODE REVIEW

File: file-formats/intf/zif_aff_types_v1.intf.abap [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
     "! Language
     language TYPE c LENGTH 2.
 
   TYPES:
-    "! ABAP language version
-    BEGIN OF ENUM abap_language_version ,
-      "! Standard
+    BEGIN OF ENUM abap_language_version,
       standard VALUE IS INITIAL,
-      "! ABAP cloud development
```
Comment (schneidermic0) [inline file-formats/intf/zif_aff_types_v1.intf.abap]: Why did you remove the ABAP doc?

**Review by schneidermic0 — CHANGES_REQUESTED** [review]
In general, the team would like to add abaplint to the project.
See my comment. Was the ABAP doc removed by mistake?

File: file-formats/intf/zif_aff_types_v1.intf.abap (hunk 1 above)
Comment (schneidermic0) [reply file-formats/intf/zif_aff_types_v1.intf.abap]: No, sorry, I missed somehow that I didn't check the final version. 
It's perfectly fine for me if the last version is correct.

**Review by schneidermic0 — APPROVED** [review]
Thanks, looks good to me. Someone of the team shall also approve

**Comment by larshp** [comment]
note that the configuration can always be changed, and is versioned like all other files in the repo

**Review by Christopher-Hermann — APPROVED** [review]
Looks good

**Comment by schneidermic0** [comment]
I have just sent a request to OSPO to enable abaplint in our repository.

**Comment by schneidermic0** [comment]
The request has already been processed by OSPO. :)
